apps/wallet-mobile/src/features/RampOnOff/common/ShowBuyBanner/BuyBannerBig.tsx

```diff
@@ -99,6 +97,7 @@ const getStyles = (props: {theme: Theme}) => {
     spaceButton: {
       paddingHorizontal: 16,
     },
+    gradientColor: theme.color.gradients['green'] as ViewStyle,
```

Comment:
to avoid this as well, which is incorrect. {'gradientColor': number} is not a StyleSheet, that's why its need to strip off only colors, or fontNames, if we are passing it down in a non-styled way.